test/integration/targets/get_url/tasks/main.yml

```diff
@@ -257,6 +257,65 @@
       - result is changed
       - "stat_result.stat.mode == '0775'"
 
+# https://github.com/ansible/ansible/issues/27617
+- name: create src file
+  copy:
+    dest: '{{ output_dir }}/27617.txt'
+    content: "ptux"
+
+- name: create sha1 checksum file of src
+  copy:
+    dest: '{{ output_dir }}/sha1sum.txt'
+    content: "a492fc1f77b49cac59b3fa65f28c37931463ebec  27617.txt"
+
+- name: set role facts
+  set_fact:
+    http_port: 27617
+    files_dir: '{{ output_dir }}/files'
+
+- name: create a directory to serve files from
+  file:
+    dest: "{{ files_dir }}"
+    state: directory
+
+- copy:
+    src: "{{ item }}"
+    dest: "{{files_dir}}/{{ item }}"
+  with_sequence: start=0 end=4 format=pass%d.json
+
+- copy:
+    src: "{{ item }}"
+    dest: "{{files_dir}}/{{ item }}"
+  with_sequence: start=0 end=30 format=fail%d.json
+
+- copy:
+    src: "testserver.py"
```

Comment:
You'll need an implementation of `testserver.py` in your PR. You should be able to base it off of the one in the `uri` integration test.

Reply:
thanks once again.